scripts/install/install-astrometry.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -ex
+
+ASTROMETRY_VERSION="${ASTROMETRY_VERSION:-0.72}"
+ASTROMETRY_DIR="astrometry.net-${ASTROMETRY_VERSION}"
+
+TAR_NAME="astrometry.net-${ASTROMETRY_VERSION}.tar.gz"
+TAR_URL="http://astrometry.net/downloads/${TAR_NAME}"
+
+ZIP_NAME="astrometry.net-${ASTROMETRY_VERSION}.zip"
+ZIP_URL="https://github.com/dstndstn/astrometry.net/archive/${ASTROMETRY_VERSION}.zip"
+
+DO_WGET="wget --dns-timeout=30 --connect-timeout=30 --read-timeout=60 --tries=1"
+
+if [ -x $PANDIR/astrometry/bin/solve-field ] ; then
+  echo "Astrometry has been cached:"
+  find $PANDIR/astrometry -type f -executable -ls
+  exit
+fi
+
+cd $PANDIR
+echo "Downloading ${TAR_URL} ..."
+$DO_WGET "${TAR_URL}" || /bin/true
+if [ -f "${TAR_NAME}" ] ; then
+  echo "Unpacking ${TAR_NAME} ..."
+  tar zxf "${TAR_NAME}"
+else
```

Comment:
If the zip is trying to work around the missing `astrometry.net` domain then we should just default to using the tarball from github. I don't see any reason to prefer the `astrometry.net` link.